docs-app/app/components/ui/form/information.gts

```diff
@@ -0,0 +1,47 @@
+import type { TOC } from '@ember/component/template-only';
+
+import strictOr from '../../../helpers/strict-or';
+import styles from './information.css';
+
+interface UiFormInformationSignature {
+  Args: {
+    formId: string;
+    instructions?: string;
+    title?: string;
+  };
+}
+
+const UiFormInformationComponent: TOC<UiFormInformationSignature> =
+  <template>
+    {{#if (strictOr @title @instructions)}}
```

Comment:
Nice, glad to see that you noticed I had created the helper `{{strict-or}}` and used that instead of the `{{or}}` from `ember-truth-helpers`.